src/app/view/components/dot-listing-data-table/dot-listing-data-table.component.html

```diff
@@ -73,16 +70,15 @@
     <td
         *ngFor="let col of columns"
         [ngStyle]="{ width: col.width, 'text-align': getAlign(col) }"
-        (click)="handleRowClick(rowData)"
     >
         <div class="listing-datatable__column-icon" *ngIf="col.icon">
             <dot-icon name="{{ col.icon(rowData) }}"></dot-icon>
             <span>{{ col.textContent || rowData[col.fieldName] }}</span>
         </div>
         <span *ngIf="!col.icon && col.fieldName !== 'nEntries'">
-            {{ rowData[col.fieldName] }}
-        </span>
-        <a
+                    {{ rowData[col.fieldName] }}
+                </span>
+        <a (click)="$event.stopPropagation()"
```

Comment:
Why stop?

Reply:
otherwise it will go edit the `content-type` instead of the `routerLink `